configure

```diff
@@ -340,6 +340,12 @@ sed -e "s;^m=none\$;m=$m;"\
     -e "s;^InstallScriptName=.*$;InstallScriptName=$installscriptname;"\
     makefiles/Mf-install.in > $w/Mf-install
 
+sed -e "s;^m=none\$;m=$m;"\
```

Comment:
This set of replacements does not properly replace the `Version=` field with the correct Chez version. However, this seems to be a bigger problem in general looking at f0c9a4fbc082fa17c5bd3bd06d82365e77edc86b, so I suppose it's fine this way.